app/src/main/java/uk/ac/ebi/atlas/experimentimport/ScxaExperimentCrud.java

```diff
@@ -72,6 +73,46 @@ public UUID createExperiment(String experimentAccession, boolean isPrivate) {
         return UUID.fromString(experimentDto.getAccessKey());
     }
 
+    @Caching(evict = {
+            @CacheEvict(cacheNames = "experiment", key = "#experimentAccession"),
+            @CacheEvict(cacheNames = "experimentAttributes", key = "#experimentAccession"),
+            @CacheEvict(cacheNames = "experiment2Collections", key="#experimentAccession"),
+            @CacheEvict(cacheNames = "speciesSummary", allEntries = true),
+            @CacheEvict(cacheNames = "jsonExperimentMetadata", key = "{#experimentAccession, 'tSnePlot'}"),
+            @CacheEvict(cacheNames = "jsonExperimentPageTabs", key = "#experimentAccession"),
+            @CacheEvict(cacheNames = "expectedClusters", key = "#experimentAccession"),
+            @CacheEvict(cacheNames = "minimumMarkerProbability", key = "#experimentAccession"),
+            @CacheEvict(cacheNames = "cellCounts", key = "#experimentAccession"),
+            @CacheEvict(cacheNames = "plotOptions", key = "#experimentAccession"),
+            @CacheEvict(cacheNames = "jsonCellMetadata", allEntries = true),
+            @CacheEvict(cacheNames = "jsonTSnePlotWithClusters", allEntries = true),
+            @CacheEvict(cacheNames = "jsonTSnePlotWithMetadata", allEntries = true),
+            @CacheEvict(cacheNames = "jsonExperimentsList", allEntries = true),
+            @CacheEvict(cacheNames = "privateExperimentAccessions", allEntries = true)})
+    public UUID createExperiment(String experimentAccession, boolean isPrivate, ExperimentType experimentType) {
```

Comment:
Remove cache annotations. We’re not goin to add experiments via the web app. In time, also, the `ScxaExperimentCrud` class will be moved to the CLI.